### PR TITLE
Fix types all-around the project

### PR DESCRIPTION
**Pull request by szepeviktor**
@mecha What do you think?

### CODE REVIEW

File: src/Cronarchy.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ class Cronarchy
      *
      * @since [*next-version*]
      *
-     * @param string     $instanceId The instance ID. Must be unique.
+     * @param int        $instanceId The instance ID. Must be unique.
```
Comment (mecha) [inline src/Cronarchy.php]: Why was this changed to an integer? The instance ID is intended to be a string, used by plugin/theme vendors to identify their instance. Did phpstan recommend this change on account of the name indicating an "ID"?

File: src/Runner.php [hunk 2]
```diff
@@ -99,7 +99,7 @@ class Runner
      *
      * @since [*next-version*]
      *
-     * @param string $instanceId The prefix to use for options in the wp_options table.
+     * @param int    $instanceId The prefix to use for options in the wp_options table.
```
Comment (mecha) [inline src/Runner.php]: This is also intended to be used as a string. See my other comment and line 247 in this file.

**Review by mecha — COMMENTED** [review]
Great changes. Just a couple of comments related to a type change.

File: src/Cronarchy.php (hunk 1 above)
Comment (szepeviktor) [reply src/Cronarchy.php]: It is contradicting, e.g. line 28: `@var int` ...

Comment (szepeviktor) [reply src/Cronarchy.php]: Done.

File: src/Runner.php (hunk 2 above)
Comment (szepeviktor) [reply src/Runner.php]: Done.

File: src/Cronarchy.php (hunk 1 above)
Comment (szepeviktor) [reply src/Cronarchy.php]: > Did phpstan recommend this change on account of the name indicating an "ID"?

No! Skynet has not taken over the globe! :)

Comment (mecha) [reply src/Cronarchy.php]: Someday :D

**Review by mecha — APPROVED** [review]
